src/libraries/System.Diagnostics.TraceSource/src/System/Diagnostics/SwitchLevelAttribute.cs

```diff
@@ -17,6 +19,7 @@ public SwitchLevelAttribute(Type switchLevelType)
         public Type SwitchLevelType
         {
             get { return _type; }
+            [MemberNotNull(nameof(_type))]
```

Comment:
I think we should add this attribute to: 
https://github.com/dotnet/runtime/blob/master/eng/ApiCompatExcludeAttributes.txt
https://github.com/dotnet/runtime/blob/master/eng/DefaultGenApiDocIds.txt

Since it can be added to public APIs that initialize private fields, like this one, so that 1) we don't generate it when running GenAPI to generate ref assemblies; 2) APICompat doesn't complaing about these attribute missing in the ref.

Reply:
Done. Thanks.